_posts/2020-03-24-Charting-non-functionals.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+authors: [peter_dekinder]
+title: 'Charting the Non-Functional Waters'
+image: /img/2020-03-24-Charting-non-functionals/nonfunc_background.jpg
+tags: [Architecture]
+category: Architecture
+comments: true
+---
+Where functional requirements are relatable to most stakeholders as they are derived from their areas of expertise, specifying the brunt of the non-functionals tends to fall to the solution architect. As these requirements are more technical in nature, the affinity of other stakeholders with them is not as pronounced. When inquiring about their relevance, the typical answer will be in very generic fashion, for example: “The application needs to be fast.” Trying to get more concrete and measurable statements is not an easy endeavor. We’ll get back to this further on in the blog post.
```

Comment:
Chapter "KPI's/quantification"?
Maybe elaborate on good metrics (#requests per second, # response time, ...)

Reply:
Het hele idee is dat er niet zoiets is als algemeen toepasbare metrics. Deze zijn sterk afhankelijk van de natuur van de oplossing. Ik ga dus geen uitspraak doen over welke goede metrics zijn. Hierover kan wel geschreven worden in het kader van een uitgewerkte solution. Misschien wel een idee om mee te nemen met een volgende architecture kata :)